### PR TITLE
Release version 4.0.0

### PR DESCRIPTION
**Pull request by jack7ellis**
### Spec 
Major version bump as changes the interfaces of the some of the classes that use the AWS SDK code. V2 code is now used.

### Description of Changes
Updated the AWS SDK dependencies (SQS and SNS) from V1 to V2. The V1 SDK packages are not compatible with newer version of Java.

### Documentation
Not Applicable.

### Risks & Impacts
Breaking change for some people. Could cause problems if a project is only using AWS SDK V1.

### Testing
Existing unit tests have been updated.

### Deployment Considerations
When this PR is merged, the new version will be uploaded and become available.

[SL-8920]: https://echobox.atlassian.net/browse/SL-8920?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: JobQueue/pom.xml [hunk 1]
```diff
@@ -152,22 +164,20 @@
         <dependency>
           <groupId>com.echobox</groupId>
           <artifactId>ebx-cachebase-sdk</artifactId>
-          <version>1.1.1</version>
+          <version>1.3.0</version>
```
Comment (dougalrea) [inline JobQueue/pom.xml:167]: Can we update to latest for cachebase-sdk, `1.4.1`?

**Review by dougalrea — COMMENTED** [review]
Based on [the release process](https://github.com/ebx/ebx-cachebase-sdk?tab=readme-ov-file#releases) outlined on ebx-cachebase-sdk (also open source) we may want to have the changelog entry included in this PR

Have also commented on cachebase-sdk version used here

File: JobQueue/pom.xml (hunk 1 above)
Comment (jack7ellis) [reply JobQueue/pom.xml:167]: Done this in a new PR: https://github.com/ebx/ebx-jobqueue-sdk/pull/26

**Review by dougalrea — APPROVED** [review]

**Review by eddspencer — APPROVED** [review]
Thanks Jack looks good! Not actually sure why we cannot merge here the git history looks good...as in there seem to not be any commits in dev that are not also in master and none of the releases to master have been squashed that I can see (the normal culprit)

**Comment by eddspencer** [comment]
Oh, I just had to remove the required `maven_validate` check that was incorrect (not actually configured to run on this branch) - now we can do it! So I will....

**Pull request merged**